### ISSUE TITLE
Room Concept for production 

### ISSUE THREAD

**Issue by utills** [tkmn0/SimpleWebRTCExample_iOS#30]
Hello,

I'm not clear how it will work for production.
So everyone will join in same server and there is no room concept as of now.
Also how will handle group video call ?
Could you please help me with these two issues ?

**Comment by tkmn0** [comment]
Hi,
first of all,  thank you for opening your issue. :)

> I'm not clear how it will work for production.
So everyone will join in same server and there is no room concept as of now.

Currently, this project focus on just p2p connection and iOS client. The signaling server is just a sample server.

 > Also how will handle group video call ?

In this case, there are some methods (eg. Full-Mesh, SFU, MCU).    
In case of full-mesh, just create peer-connections x (number of participants - 1). Then exchange signaling messages for each peer-connections.
Actually, I created full-mesh signaling application for web client named [peers](https://github.com/tkmn0/peers-server). So, in the feature, I will create sample peers client application for iOS! :) 

In case of SFU, I created SFU server application written in typescript, [here](https://github.com/tkmn0/medooze-sfu-typescript).  However, I will not create SFU client for iOS because SFU is costly for server side.

Thanks again :)
